toc.yml

```diff
@@ -0,0 +1,5 @@
+- name: MsQuic
```

Comment:
Can these new files be put in the `doc` folder instead of the root?

Reply:
I don't think so, because the docfx engine uses directory structure to build the site. Each subfolder has a toc.yml to define its manifest. This toc.yml file is the root manifest.

Reply:
I very strongly don't want any more files at the root of the repo. We need to keep things as clean as possible. Most people will never need to look at these files in the repo iteself (instead use the doc website).

I assume we should be able to have an extra step when generating the docfx directory structure that JIT moves things where we want them.

Reply:
Ahh I see, I understand the concern. JIT sounds good!